content/authorization/authorizing-api-calls/index.mdx

```diff
@@ -764,6 +779,126 @@ curl "https://accounts.livechat.com/customer/" \
+| `client_id`     | yes      | Client ID of your application                                                                                                                                                                                                                                                                      |
+| `response_type` | yes      | Value: `token`                                                                                                                                                                                                                                                                                     |
+| `entity_id`     | no       | Customer ID. If you don't specify it, a new customer will be created (Step 1). Specify it (described in Step 2) if you want to [acquire a new token for an existing customer](/getting-started/authorization/faq/#the-access-token-of-my-customer-has-expired-do-i-need-to-create-a-new-customer). |
+
+#### Response
+
+The response is a JSON with the following parameters:
+
+| Parameter      | Description                                                                                                                                                                                                                                                      |
+| -------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `access_token` | A token you can use to call LiveChat APIs on behalf of the user (customer).                                                                                                                                                                                      |
+| `client_id`    | **Client Id** of your app                                                                                                                                                                                                                                        |
+| `entity_id`    | The ID of the newly created customer                                                                                                                                                                                                                             |
+| `expires_in`   | A number in seconds specifying how long the `access_token` will be valid; 28800 sec by default. When it expires, you will need to generate new `access_token` using `refresh_token` (read [Using the refresh token](#using-the-refresh-token) for more details). |
```

Comment:
```suggestion
| `expires_in`   | A number in seconds specifying how long your customer `access_token` will be valid; 28800 sec by default. When it expires, you will need to generate new `access_token` using `refresh_token` (read [Using the refresh token](#using-the-refresh-token) for more details). |
```

Reply:
There was a little mistake here, I changed it to:

A number in seconds specifying how long the `access_token` will be valid; 28800 sec by default. When it expires, you will need to generate new `access_token` following [Case: Existing customer](#case-existing-customer-1)).